dapps/react-dapp-v2/next.config.js

```diff
@@ -3,6 +3,14 @@ const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
   distDir: "build",
+  webpack(config) {
```

Comment:
Hey, this config is currently needed because webpack no longer includes polyfills for node.js core modules by default

The fs module is used in a dependency of `@elrondnetwork/erdjs-walletcore` (`@elrondnetwork/bls-wasm`) that's not currently used in code, so there's no functionality impact. 
We are working on extracting the packages causing the issue to separate repositories, but this might take a while.

We will update the config file once we have the new version.

Reply:
Gotcha, thanks for clarifying 👍